modules/sde-core/src/main/java/org/eclipse/tractusx/sde/core/processreport/model/ProcessFailureDetails.java

```diff
@@ -0,0 +1,41 @@
+/********************************************************************************
+ * Copyright (c) 2022 T-Systems International GmbH
```

Comment:
please add 2023 year in copyright header

Reply:
Copyright header added.